tests/storage/zfs_ng/test_zfs_sr_crosspool_migration.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from tests.storage import cold_migration_then_come_back, live_storage_migration_then_come_back
+
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host >= 8.2 with an additional unused disk for the SR.
```

Comment:
Depend on `host_at_least_8_3`?
```suggestion
# - host(A1): first XCP-ng host >= 8.3 with an additional unused disk for the SR.
```

(Same for `test_zfs(_ng)_sr_intrapool_migration

Reply:
applied at module level in #202